src/main/frontend/components/page.cljs

```diff
@@ -1041,15 +1041,15 @@
          [:div.r.flex.items-center.justify-between
           [:div
            (ui/tippy
-            {:html  [:small (str (t :page/show-whiteboards) " ?")]
```

Comment:
`?` ? 😆

Reply:
Forgot to bring in the old comments. See https://github.com/logseq/logseq/pull/8722#discussion_r1122083939
Punctuation should be part of the translation (also see https://github.com/logseq/logseq/pull/8722#discussion_r1122081808 that didn't make it into this PR). The question mark is superfluous in this case, so removing it is a better option in my opinion, instead of invalidating the translations to include it.

Reply:
Agree with this change. Just thought the original was funny.
 
> Punctuation should be part of the translation

Agree with '?' and '!' but what about delimiting characters like ':' or '-'? Moving those characters into translation will likely make it easier to forget to delimit in some languages. Grep for `'(str (t'` for examples. Whatever we decide we should document in the translation guide and if possible remove counter examples

Reply:
> Forgetting to delimit could happen, but having the ability to properly translate the app should be the top priority in my opinion

Seems reasonable. Could you remove some of the counter examples from the above grep and document the punctation guidance?

Reply:
Replaced all `(str (t` instances and invalidated the previous translations . We still have to fix some remaining cases (grep `" (t `). When all of these are fixed, it might be a good idea to update our linter. i also updated dev-practices.md.